blocks/default-output-block/output-types/default.jsx

```diff
@@ -195,4 +196,20 @@ const SampleOutputType = ({
   );
 };
 
+// if publisher wants no sites to be spa
+// then they do nothing. spaSites will be falsy undefined and fallsback to spa true,
+//  which won't do anything in isolation
+
+// if publisher wants all sites to be spa
+// then they set environment "FUSION_SERVICE_WORKER": true and use 2.8
+//  spaSites will be falsy undefined and fallback to spa true
+
+// if publisher wants to select which sites are spa
+// then set environment "FUSION_SERVICE_WORKER": true and use 2.8
+//    and set in blocks.json spaSites: ["target-site-id"].
+//    spaSites will be a truthy array and set itself
+
+// fallback to true to ensure all site ids don't have to copy-pasted to blocks.json
+SampleOutputType.spa = blocks.spaSites ? blocks.spaSites : true;
```

Comment:
What happens if `blocks.spaSites` is a blank array? Will Fusion treat that as a true or false?

Reply:
true

Reply:
And they would effectively have no spaSites supported

Reply:
I was more curious to know what Fusion would do and how we document what values can be in blocks.json `spaSites`. Also noting that it's possible that the key may always be present but not contain anything

Reply:
Can we add tests for this to ensure each permutation is clear

Reply:
updated to make sure the expected behavior is reflected in the tests